projects/dsp-ui/src/lib/viewer/services/event-bus.service.ts

```diff
@@ -0,0 +1,33 @@
+import { Injectable } from '@angular/core';
+import { Subject, Subscription } from 'rxjs';
+import { filter, map } from 'rxjs/operators';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class EventBusService {
```

Comment:
Some jsdocs description should be added to explain the methods of the service

Reply:
added in c212919